apps/expo/src/components/EventTab.tsx

```diff
@@ -12,17 +15,75 @@ interface EventProps {
 }
 
 export default function EventTab({ activity, attending }: EventProps) {
+  const [isAttending, setIsAttending] = useState(attending);
+  const utils = api.useContext();
```

Comment:
This is deprecated and will be removed in future trpc versions, maybe you can use `useUtils`?